dashboard-frontend/src/__tests__/user.spec.tsx

```diff
@@ -46,48 +62,81 @@ describe('user', () => {
     expect(login).toHaveBeenCalledTimes(1);
   });
 
-  it('authenticated user accessing login page, are redirected to studies page', () => {
-    userAuthenticated(true);
-
-    visit('/login');
-
-    screen.getByTestId('studies-page');
-  });
-
-  it('authenticated users can see their name/email and avatar in the header', () => {
-    const user = {
-      name: 'testuser',
-      avatarUrl: 'https://s.img.com/avatar.png',
-    };
-    userAuthenticated(true, user);
-
-    visit('/');
-
-    const header = screen.getByTestId('header');
-    expect(header).toContainElement(screen.getByText(user.name));
-    const userAvatar = screen.getByTestId('user-avatar');
-    expect(header).toContainElement(userAvatar);
-    expect(userAvatar).toHaveAttribute('src', user.avatarUrl);
-  });
-
-  it('clicking user avatar opens a dropdown with a Sign out button', () => {
-    userAuthenticated(true);
-    visit('/');
-
-    userEvent.click(screen.getByTestId('user-avatar'));
-
-    screen.getByText('Sign out');
-  });
-
-  it('clicking the Sign out button, starts the logout flow', () => {
-    const logout = jest.fn();
-    userAuthenticated(true, { logout });
-    visit('/');
-    userEvent.click(screen.getByTestId('user-avatar'));
-
-    userEvent.click(screen.getByText('Sign out'));
-
-    expect(logout).toHaveBeenCalledTimes(1);
+  describe('when is authenticated', () => {
+    it('the app tries to create the user in our backend, when it fails for a reason other than "user already exists", the login page with a generic error is shown', async () => {
+      userAuthenticated(true);
+      server.post('/users', () => new Response(500));
+
+      visit('/');
+      screen.getByTestId('loading-page');
+
+      await waitFor(() => {
+        screen.getByText('Oops, something went wrong!');
+        screen.getByText(
+          'Please check your internet connection and try again.'
```

Comment:
I would say show the message from the server, and if there is none, just show "500: server error". It's annoying to the user, but that's the backend developers fault for not sending helpful messages!

Reply:
donde "error.message" es, o del response o del http client.

Reply:
Yes makes sense, it will be more useful 👍

This change may require update the layout code for certain resolutions, I'll handle them in the following ticket: https://github.com/vlab-research/vlab/issues/77